actions/release/1.0/README.md

```diff
@@ -338,4 +338,6 @@ release action 运行成功后会在当前目录生成 `dicehub-release` 文件
             registryUsername: ${{ secrets.REGISTRY_USERNAME }}
             # 远端镜像仓库的密码
             registryPassword: ${{ secrets.REGISTRY_PASSWORD }}
+            # 制品 不使用retag中镜像地址
+            notRetag: true
```

Comment:
`notRetag` points to ambiguity, what is needed is the replacement of the image in the dice release, rather than determining whether to execute a retag. @CeerDecy

Reply:
have change to `releaseImgNotRetag`